test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/common/common.py

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+from api.cas import casadm
+from api.cas.cache_config import KernelParameters, CacheLineSize, CleaningPolicy, CacheMode
+
+
+def test_prepare(cache_device, cache_mode: CacheMode = None, cache_line_size: CacheLineSize = None,
+                 kernel_params: KernelParameters = KernelParameters(), cleaning_policy: CleaningPolicy = None):
+    #  Change cleaning policy to default for Write Policy different than WB
+    if cleaning_policy:
+        cleaning_policy = CleaningPolicy.DEFAULT if cache_mode != CacheMode.WB \
+            else cleaning_policy
```

Comment:
Assigning `CleaningPolicy.DEFAULT` as the default value of `cleaning_policy` param would allow you to get rid of these three lines